src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -537,6 +549,10 @@ type KafkaAPI struct {
 	External ExternalConnectivityConfig `json:"external,omitempty"`
 	// Configuration of TLS for Kafka API
 	TLS KafkaAPITLS `json:"tls,omitempty"`
+	// AuthenticationMethod can enable authentication method per Kafka
+	// listener. Available options are: none, sasl, mtls_identity.
+	// https://docs.redpanda.com/docs/security/authentication/
+	AuthenticationMethod string `json:"authenticationMethod,omitempty"`
```

Comment:
I would like to see webhook validation for the allowed values in AuthenticationMethod 🙏

Reply:
Ok will add